packages/appstore/src/components/modals/wallet-dummy-success/wallet-dummy-success.tsx

```diff
@@ -0,0 +1,126 @@
+import { useHistory } from 'react-router';
+import { Button, Modal, Text, DesktopWrapper, MobileDialog, MobileWrapper } from '@deriv/components';
+import { routes, formatMoney } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletAppCard from 'Components/wallet-app-card';
+import { getAccountName } from 'Constants/utils';
+import { useActiveWallet } from '@deriv/hooks';
+import './wallet-dummy-success.scss';
+
+const WalletDummySuccess = observer(() => {
+    const { ui, traders_hub } = useStore();
+    const { is_wallet_create_new_account_modal, setWalletCreateNewAccountModal } = traders_hub;
+
+    const { is_mobile } = ui;
+    const active_wallet = useActiveWallet();
+
+    const account_title = `${getAccountName({
+        account_type: 'trading',
+    })} (${active_wallet?.landing_company_name?.toUpperCase()})`;
+    const account_balance = formatMoney(active_wallet?.currency_config?.display_code, active_wallet?.balance, true);
+
+    const currency_title = getAccountName({
+        account_type: 'wallet',
+        display_currency_code: active_wallet?.currency_config?.display_code,
+    });
+
+    const toggleWalletCreateNewAccountModal = () => {
+        setWalletCreateNewAccountModal(!is_wallet_create_new_account_modal);
+    };
+
+    const wallet_details = {
+        account_title,
+        balance: account_balance,
+        currency_title,
+        currency: active_wallet?.currency_config?.display_code,
+        gradient_card_class: active_wallet?.gradient_card_class,
+        icon: active_wallet?.icon,
+        is_demo: active_wallet?.is_demo,
+    };
+
+    const history = useHistory();
+
+    const ModalContent = () => (
+        <React.Fragment>
+            <div className='wallet-success-dialog__icon'>
+                <WalletAppCard wallet={wallet_details} />
+            </div>
+            <Text as='h2' size={is_mobile ? 'xs' : 's'} weight='bold' className='wallet-success-dialog__title'>
+                <Localize i18n_default_text='Your {{account_title}} account is ready' values={{ account_title }} />
+            </Text>
+            <Text
+                as='p'
+                size={is_mobile ? 'xxs' : 'xs'}
+                line_height='s'
+                align='center'
+                className='wallet-success-dialog__description'
+            >
+                <Localize
+                    i18n_default_text='Transfer funds from your {{currency_title}} to your {{account_title}} account to start trading.'
+                    values={{
+                        account_title,
+                        currency_title,
+                    }}
+                />
+            </Text>
+        </React.Fragment>
+    );
+
+    const ModalFooter = () => (
+        <Button.Group>
+            <Button secondary onClick={() => setWalletCreateNewAccountModal(false)}>
+                {localize('Maybe later')}
```

Comment:
Pls use the component

Reply:
For button text it seems like it is using some sort of function. My guess is to capitalise the first letter of the string. It seems that we can't simply pass the `<Localize />` component.
```
{text &&
                !(is_loading || is_submit_success) &&
                ((typeof renderText === 'function' && renderText(text[0].toUpperCase() + text.substr(1))) || (
                    <Text size='xs' weight='bold' align='center' className={classNames('dc-btn__text', classNameSpan)}>
                        {text[0].toUpperCase() + text.substr(1)}
                    </Text>
                ))}
```

Reply:
That is for text Prop but i think we can pass Localize as children.

Reply:
@arshad-rao-deriv from the previous card I was trying to make the same attempt as the one suggested by @thisyahlen-deriv but it was returning me some error. Regardless, will try this once more to see if the issue is still there, if not then I will continue with the implementation. Thanks for the reply 🙏🏼